core/common/src/ConcurrentQuery.ts

```diff
@@ -45,12 +45,19 @@ export interface QueryLimit {
 
 /** @public */
 export interface QueryPropertyMetaData {
+  /** The class name is set to empty if the property is a generated one, otherwise, it is the name of the ECClass that the property is contained within. */
   className: string;
+  /** Access string is the property's display label if the property is a generated one, otherwise it is the ECSQL property path. */
```

Comment:
is 'display label' the right term to use here?  Is this the alias?

Reply:
Yes, "display label" is the correct term to use.
You can see how it's implemented in the code here: https://github.com/iTwin/imodel-native/blob/main/iModelCore/ECDb/ECDb/QueryJsonAdaptor.cpp#L371. This approach mirrors the one used in the `ECSqlColumnInfo.getAccessString()` function, as seen here: https://github.com/iTwin/imodel-native/blob/main/iModelJsNodeAddon/IModelJsNative.cpp#L3663-L3682

Reply:
Updated the document to use `alias` instead of `display label`